maps/site53/job/jobs/misc.dm

```diff
@@ -63,6 +65,7 @@
 	)
 
 	minimal_access = list()
+	skill_points = 30 //its a office worker, gets more points than literally anyone else
```

Comment:
```suggestion
	skill_points = 20
```
Holy crap dude, THIRTY points? Also, bad form

Reply:
Possibly even less, doesn't make sense for an O.W. to be trained in a lot of skills.